APIv2/articles.md

```diff
@@ -691,7 +691,7 @@ Body: List of [PrivateFilePresenter](presenters/file.md#privatefilepresenter)
 |Name               |Type                   |Description                                |
 |-------------------|-----------------------|-------------------------------------------|
 |`md5`              |`str`                  |MD5 sum pre computed on the client side|
-|`name`              |`str`                  |File name including the exception|
+|`name`              |`str`                  |File name including the extention|
```

Comment:
Tom, thanks for opening this. I think it's `extension` though. Are you ok to update your PR so we can merge it in?